stdlib/public/Darwin/Accelerate/vForce_Operations.swift

```diff
@@ -0,0 +1,1781 @@
+        V : _MutableContiguousCollection,
+        U.Element == Double, V.Element == Double {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvcos(dest.baseAddress!,
+                          src.baseAddress!,
+                          &n)
+                }
+            }
+    }
+    
+    /// Calculates the cosine of pi multiplied by each element in `vector`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func cosPi<U, V>(_ vector: U,
+                            result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Float, V.Element == Float {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvcospif(dest.baseAddress!,
+                             src.baseAddress!,
+                             &n)
+                }
+            }
+    }
+    
+    /// Calculates the cosine of pi multiplied by each element in `vector`, writing the result to `result`, double-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func cosPi<U, V>(_ vector: U,
+                            result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Double, V.Element == Double {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvcospi(dest.baseAddress!,
+                            src.baseAddress!,
+                            &n)
+                }
+            }
+    }
+    
+    // MARK: Cosine & Sine
+    
+    /// Calculates the sine and cosine of each element in `vector`, writing the results to `sinresult` and `cosresult` respectively, single-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter sinresult: Output sine values.
+    /// - Parameter cosresult: Output cosine values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func sinCos<T, U, V>(_ vector: T,
```

Comment:
I think this should either use the term of art `sincos` or be a little more explicit: `sinAndCos` or similar.

Reply:
changed to `sincos`.